SECURITY.md

```diff
@@ -10,5 +10,6 @@
 | 3.30    | 27/12/2024      |
 | 3.31    | Best effort     |
 | 3.32    | Best effort     |
+| 3.33    | Best effort     |
 
 ### Version 3.23 is prematurely end of support because its staging build system was no longer working.
```

Comment:
Tu peux aussi enlever cette ligne, his initial support was for 2021: https://github.com/mapfish/mapfish-print/commit/34a173559786c18706a293d6cd0a5d7c993a6ad2